packages/core-data/src/entities.js

```diff
@@ -207,6 +210,15 @@ export const rootEntitiesConfig = [
 		baseURLParams: { context: 'edit' },
 		plural: 'globalStylesVariations', // Should be different than name.
 		getTitle: ( record ) => record?.title?.rendered || record?.title,
+		getRevisionsUrl: ( parentId, revisionId ) =>
+			`/wp/v2/global-styles/${ parentId }/revisions${
+				revisionId ? '/' + revisionId : ''
+			}`,
+		revisionURLParams: { context: 'view' },
```

Comment:
Do we need this config or can't we just add it to the `getRevisionsUrl` function?

Reply:
> can't we just add it to the getRevisionsUrl function?

~~You bet we can! I was just following the `baseURL` pattern.~~

Actually, it looks like `view` is the default anyway. 

https://developer.wordpress.org/rest-api/reference/post-revisions/#arguments

I can just remove it.